2d/src/main/java/org/tweetwallfx/twod/TagTweets.java

```diff
@@ -149,15 +132,9 @@ public void start() {
 
         try {
             System.out.println("** 1. Creating Tag Cloud for " + searchText);
-            Query query = new Query(searchText);
-            query.setCount(100);
-            twitter = new TwitterFactory(conf).getInstance();
-            QueryResult result = twitter.search(query);
-
-            buildTagCloud(result.getTweets());
+            buildTagCloud(tweeter.search(new TweetQuery().query(searchText).count(100)));
             createWordle();
-
-        } catch (TwitterException ex) {
+        } catch (TweetException ex) {
```

Comment:
Exception handling enhanced. Logging/NIL-behavior and with the ability to observe the latest exception